src/components/Header.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  
+  <link rel="stylesheet" href="../styles/root.css">
+</head>
+<body>
+<!-------------------------Header Page Start------------------------->
+  
+<div class="ComponentMainBoxOne img-responsive">
+
+  <header class="headerContainerMain header-center Header-TextSize box-shadow quiz-container-bg">
+    <div class="headerContainerOne flex-evenly padding-small-left1">
+      <img class="logoImage" src="./Images/Phoenix UI.jpg" alt="">
```

Comment:
provide alt attribute some meaningful value

Reply:
Okay, I'll do that.